src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -72,9 +72,14 @@ object Import {
       """Declares endpoints. The default is Map("web" -> Endpoint("http", 0, Set("http://:9000"))) where the service name is the name of this project. The "web" key is used to form a set of environment variables for your components. For example you will have a `WEB_BIND_PORT` in this example."""
     )
 
+    val checkInitialDelay = SettingKey[Int](
```

Comment:
Simply round to the second. 

> On 5 Aug 2015, at 8:12 am, Markus Jura <notifications@github.com> wrote:
> 
> In src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala:
> 
> > @@ -72,9 +72,14 @@ object Import {
> >        """Declares endpoints. The default is Map("web" -> Endpoint("http", 0, Set("http://:9000"))) where the service name is the name of this project. The "web" key is used to form a set of environment variables for your components. For example you will have a `WEB_BIND_PORT` in this example."""
> >      )
> >  
> > +    val checkInitialDelay = SettingKey[Int](
> Ok, will use FiniteDuration.
> 
> So far the check command only takes a value in seconds, not more fine-grained value such as milliseconds. What should we do if the user specifies 500.milliseconds?
> 
> Should we change the check.py logic so that it takes the --initial-delay as milliseconds as well or should we do something else?
> 
> —
> Reply to this email directly or view it on GitHub.
>